test/1_stdlib/PrintInteger.swift

```diff
@@ -33,9 +33,9 @@ PrintTests.test("CustomStringConvertible") {
   hasDescription(CInt(42))
   hasDescription(CLong(42))
   hasDescription(CLongLong(42))
-  hasDescription(CWideChar(42))
+  hasDescription(CWideChar(42)!)
```

Comment:
I think it will treat it as Int. I had to do hasDescription(CWideChar(UInt8(42))) to get rid of the unwrapping.